CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## [0.18.4](https://github.com/eclipse/lemminx/milestone/26?closed=1) (February 14, 2022)
+
+### Enhancements
+
+- Eliminate need for downloading just-released binary checksums.. See [#650](https://github.com/redhat-developer/vscode-xml/pull/650).
+- Prevent suspicious directory traversal. See [lemminx#1171](https://github.com/eclipse/lemminx/pull/1171).
+- Basic support for parameter entities. See [lemminx#1167](https://github.com/eclipse/lemminx/issues/1167).
+- Support for document link DTD entity SYSTEM. See [lemminx#1165](https://github.com/eclipse/lemminx/issues/1165).
+
+### Bug Fixes
+
+- Bad SYSTEM for DTD DocType and Entity breaks the XML validation. See [lemminx#1169](https://github.com/eclipse/lemminx/issues/1169).
```

Comment:
Can we use the bug on the vscode-xml side for this ? Eg. https://github.com/redhat-developer/vscode-xml/issues/647 . You can always keep the title the same or change to explain better than the original bug.